libraries/chain/database.cpp

```diff
@@ -352,6 +352,15 @@ namespace golos { namespace chain {
             return _clear_votes_block > head_block_num();
         }
 
+        void database::set_accounts_to_store_metadata(const std::vector<std::string>& accounts_to_store_metadata) {
+            _accounts_to_store_metadata = accounts_to_store_metadata;
+        }
+
+        bool database::store_metadata_for_account(const std::string& name) const {
+            auto& v = _accounts_to_store_metadata;
+            return std::find(v.begin(), v.end(), name) != v.end();
```

Comment:
It is impossible to store metadata for all users.

It is not useful for full API nodes.